configure.ac

```diff
@@ -49,6 +49,14 @@ AC_ARG_VAR([OMR_CROSS_CONFIGURE],
 	Compiler and feature detection tests will be invalid. All options must be configured manually.
 	One of: yes,no. (Default: no)])
 
+AC_ARG_VAR([OMR_PRODUCT_DESCRIPTION], [The description of product integerating OMR. (Default: OMR 1.0)])
+AC_ARG_VAR([OMR_PRODUCT_NAME], [The name of product integerating OMR. (Default: OMR)])
+AC_ARG_VAR([OMR_PRODUCT_VERSION], [The version of product integerating OMR. (Default: 1.0)])
+
+OMR_PRODUCT_DESCRIPTION=${OMR_PRODUCT_DESCRIPTION:-"AC_PACKAGE_STRING AC_PACKAGE_URL"}
```

Comment:
Where do `AC_PACKAGE_STRING AC_PACKAGE_URL', `AC_PACKAGE_NAME` and `AC_PACKAGE_VERSION` come from and what are their values?